independent-projects/bootstrap/app-model/src/main/java/io/quarkus/paths/ArchivePathTree.java

```diff
@@ -251,7 +252,7 @@ protected void initMultiReleaseMapping(Map<String, String> mrMapping) {
 
         @Override
         public boolean isOpen() {
-            return fs.isOpen();
+            return fs != null && fs.isOpen();
```

Comment:
I'd acquire a read lock for this check too

Reply:
Oh, yes, you're right, silly mistake!